README.md

```diff
@@ -1 +1,338 @@
 ![travis ci](https://travis-ci.org/electrode-io/react-native-electrode-bridge.svg?branch=master)
+
+
+Electrode React Native Bridge
+-----------------------------
+
+This project is essentially a react native library, consisting of a JavaScript module and an associated Android Native Module and iOS Native module.
+
+It is built on top of the react native built-in bridging constructs (to communicate between the react native JS side and Native side) and offers a clean bi-directional communication API, exposing methods to send events and/or requests from/to any side of the bridge (JS/Native).
+It offers more options and flexibility to communicate between the JS/Native side that is not offered out of the box by react native. Ultimately it can help with integrating react native applications into existing native code bases.
```

Comment:
"It offers more options and flexibility to communicate between the JS/Native side that is not offered out of the box by react native." We are listing if down, may not be needed.

Reply:
It is built on top of react native built-in bridging constructs (to communicate between react native and pure Native) and offer a clean bi-directional communication API, exposing methods to send events and/or requests from/to any side of the bridge (JS/Native).

It offers more options and flexibility to communicate between React Native and pure Native (iOS/Android). Ultimately it can help with integrating react native applications into existing native apps.

second point really not required as we are talking about the bridge and not about the platform